<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>6</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCustomPlot" name="chart_1" native="true"/>
           </item>
           <item>
            <widget class="QCustomPlot" name="chart_3" native="true"/>
           </item>
           <item>
            <widget class="QCustomPlot" name="chart_2" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Sensor Properties</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="MyGLWidget" name="myGLWidget" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>X min</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>X max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_4">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>16</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="value">
            <number>360</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Y min</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderYMin">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>16</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Y max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderYMax">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>16</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="value">
            <number>360</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Z min</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderZMin">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>16</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Z max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderZMax">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>16</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="value">
            <number>360</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>horizontalSlider_4</zorder>
      <zorder>sliderZMax</zorder>
      <zorder>sliderYMax</zorder>
      <zorder>label</zorder>
      <zorder>label_2</zorder>
      <zorder>label_3</zorder>
      <zorder>label_4</zorder>
      <zorder>label_5</zorder>
      <zorder>label_6</zorder>
      <zorder>sliderYMin</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionActionConnect"/>
   <addaction name="actionActionDisconnect"/>
   <addaction name="actionActionConfigure"/>
   <addaction name="actionStartRecording"/>
   <addaction name="actionPauseRecording"/>
   <addaction name="actionStopRecording"/>
  </widget>
  <action name="actionActionConnect">
   <property name="icon">
    <iconset resource="images/resources.qrc">
     <normaloff>:/connect.png</normaloff>:/connect.png</iconset>
   </property>
   <property name="text">
    <string>actionConnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
  </action>
  <action name="actionActionDisconnect">
   <property name="icon">
    <iconset resource="images/resources.qrc">
     <normaloff>:/disconnect.png</normaloff>:/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>actionDisconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect serial port</string>
   </property>
  </action>
  <action name="actionActionConfigure">
   <property name="icon">
    <iconset resource="images/resources.qrc">
     <normaloff>:/settings.png</normaloff>:/settings.png</iconset>
   </property>
   <property name="text">
    <string>actionConfigure</string>
   </property>
   <property name="toolTip">
    <string>Configure the seriap port</string>
   </property>
  </action>
  <action name="actionStartRecording">
   <property name="icon">
    <iconset resource="images/resources.qrc">
     <normaloff>:/play_icon.png</normaloff>:/play_icon.png</iconset>
   </property>
   <property name="text">
    <string>startRecording</string>
   </property>
   <property name="toolTip">
    <string>start measuring</string>
   </property>
  </action>
  <action name="actionPauseRecording">
   <property name="icon">
    <iconset resource="images/resources.qrc">
     <normaloff>:/pause_icon.png</normaloff>:/pause_icon.png</iconset>
   </property>
   <property name="text">
    <string>pauseRecording</string>
   </property>
   <property name="toolTip">
    <string>pause mesuring</string>
   </property>
  </action>
  <action name="actionStopRecording">
   <property name="icon">
    <iconset resource="images/resources.qrc">
     <normaloff>:/stop_icon.png</normaloff>:/stop_icon.png</iconset>
   </property>
   <property name="text">
    <string>stopRecording</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
   <slots>
    <slot>valueChangedSlot(int)</slot>
    <slot>valueChangedSlotMax(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header>myglwidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setYRotation(int)</slot>
    <slot>setZRotation(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>sliderYMin</tabstop>
  <tabstop>sliderZMin</tabstop>
 </tabstops>
 <resources>
  <include location="images/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>chart_1</receiver>
   <slot>valueChangedSlot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderYMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>chart_3</receiver>
   <slot>valueChangedSlot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderZMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>chart_2</receiver>
   <slot>valueChangedSlot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>549</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setXRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>812</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderYMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setYRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>812</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderZMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setZRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>812</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>chart_1</receiver>
   <slot>valueChangedSlotMax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>726</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderYMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>chart_3</receiver>
   <slot>valueChangedSlotMax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>702</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>546</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderZMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>chart_2</receiver>
   <slot>valueChangedSlotMax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
